src/content/post/2025-11-28-92fce5fc.md

```
@@ -0,0 +1,78 @@
+```
+
+普通にcliから実行すると値がでてくるので、おそらく`go`から叩くとなにか認証とかが上手くとおってないのではないだろうか?
+
+### goplsの設定
+
+手元でgoplsの補完(たとえば`bytes.Buffer`)をしたとき、`bytes.`の段階で補完すると`bytes.bytes.Buffer`になってしまう。
+
+`iskeyword`に`.`が入ってないのでうまくいかない。
+
+```help
+						*'iskeyword'* *'isk'*
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)